backends/aptcc/apt-intf.cpp

```diff
@@ -1743,7 +1743,7 @@ void AptIntf::updateInterface(int fd, int writeFd, bool *errorEmitted)
             const gchar *status   = g_strstrip(split[0]);
             const gchar *pkg      = g_strstrip(split[1]);
             const gchar *percent  = g_strstrip(split[2]);
-            g_autofree gchar *str = g_strdup(g_strstrip(split[3]));
+            std::string const str = g_strstrip(split[3]);
```

Comment:
Mind changing to `const std::string` matching the rest of the style

Reply:
Good advice on the rest of the style! Pushed a new revision.